From e2392944cc779b2305bdbb7e6b9d9bb57d5398ee Mon Sep 17 00:00:00 2001
From: Lee Duncan <lduncan@suse.com>
Date: Tue, 9 Jun 2020 11:22:50 -0700
Subject: [PATCH] Handle NIC names longer than 7 characters.

The output of "fcoeadm -s" assumed a short NIC name,
so handle up to 15 characters, the current standard.
---
 fcoeadm_display.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fcoeadm_display.c b/fcoeadm_display.c
index f7e6c43806ad..7d29422e91fa 100644
--- a/fcoeadm_display.c
+++ b/fcoeadm_display.c
@@ -455,7 +455,7 @@ static void scan_device_map(char *rport, enum disp_style style)
 static void show_port_stats_header(const char *ifname, int interval)
 {
 	printf("\n");
-	printf("%-7s interval: %-2d                                    Err  Inv  "
+	printf("%-15s interval: %-2d                            Err  Inv  "
 	       "IvTx Link Cntl Input     Input     Output    Output\n",
 	       ifname, interval);
 	printf("Seconds TxFrames  TxBytes      RxFrames  RxBytes        "
@@ -893,7 +893,7 @@ static void
 print_interface_fcoe_lesb_stats_header(const char *ifname, int interval)
 {
 	printf("\n");
-	printf("%-7s interval: %-2d\n", ifname, interval);
+	printf("%-15s interval: %-2d\n", ifname, interval);
 	printf("LinkFail VLinkFail MissFKA SymbErr ErrBlkCnt FCSErrCnt\n");
 	printf("-------- --------- ------- ------- --------- ---------\n");
 }
-- 
2.26.2

